plugins/woocommerce-admin/client/guided-tours/variable-product-tour/index.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+import { useUserPreferences } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+
+function getStepName(
+	steps: TourKitTypes.WooStep[],
+	currentStepIndex: number
+) {
+	return steps[ currentStepIndex ]?.meta?.name;
+}
+
+export const VariableProductTour = () => {
+	const [ isTourOpen, setIsTourOpen ] = useState( false );
+
+	const { updateUserPreferences, variable_product_tour_shown: hasShownTour } =
+		useUserPreferences();
+
+	const config: TourKitTypes.WooConfig = {
+		steps: [
+			{
+				referenceElements: {
+					desktop: '.attribute_tab',
+				},
+				focusElement: {
+					desktop: '.attribute_tab',
+				},
+				meta: {
+					name: 'attributes',
+					heading: __( 'Start by adding attributes', 'woocommerce' ),
+					descriptions: {
+						desktop: __(
+							'Add attributes like size and color for customers to choose from on the product page. We will use them to generate product variations.',
+							'woocommerce'
+						),
+					},
+					primaryButton: {
+						text: __( 'Got it', 'woocommerce' ),
+					},
+				},
+			},
+		],
+		options: {
+			callbacks: {
+				onNextStep: ( currentStepIndex ) => {
+					recordEvent( 'variable_product_tour_step_viewed', {
+						step: getStepName( config.steps, currentStepIndex + 1 ),
+					} );
```

Comment:
Same as [the other comment](https://github.com/woocommerce/woocommerce/pull/37413/files#r1147975084).

Reply:
Removed in e2e2875